src/pages/contact.tsx

```diff
@@ -7,7 +7,97 @@ const ContactPage: React.FC<PageProps> = () => {
   return (
     <Layout>
       <SEO />
-      {/** TODO */}
+      <div className="site-section ftco-subscribe-1 site-blocks-cover pb-4" style={{ backgroundImage: `url('/images/bg_3.jpg')` }}>
+        <div className="container">
+          <div className="row align-items-end">
+            <div className="col-lg-7">
+              <h2 className="mb-0">Kapcsolat</h2>
+              <p>Elérhetőség, térkép és fényképek</p>
+            </div>
+          </div>
+        </div>
+      </div>
+
+      <div className="custom-breadcrumbs border-bottom">
```

Comment:
There is a typo, should be  `custom-breadcrumns` OR should fix typo in *_site-blocks.scss*

Reply:
Oh jesus, so that was the problem! thx